### DISCUSSION TITLE
NTC-Templates version that is compatible with textfsm ver 0.4.1

### DISCUSSION THREAD

**Discussion by RepotSirc** [networktocode/ntc-templates#1364] [General]
Hi,

We have strict company restrictions that only allows us to use python 3.6, due to this I am only able to install netmiko 2.3.3 and textfsm 0.4.1. And I just need to know which ntc-templates version is compatible with textfsm 0.4.1.

Thanks in advance.

**Comment by itdependsnetworks** [comment]
We use standard Python mechanisms such as setup.py and pyproject.toml and you can follow the the version requirements via tags: https://github.com/networktocode/ntc-templates/tags

**Reply by mjbear** [reply]
@RepotSirc
Did you find an answer to this?

From what I can see based on the pyproject.toml tag 1.7.0 is the last one not tagged with requiring textfsm 1.1.0.
https://github.com/networktocode/ntc-templates/blob/v1.7.0/pyproject.toml

Whereas 2.0.0 has requirements for textfsm 1.1.0, but only requires Python 3.6.
https://github.com/networktocode/ntc-templates/blob/v2.0.0/pyproject.toml#L16

And 3.1.0 is the last release specifying Python 3.6 (but like 2.0.0 requires textfsm 1.1.0)
https://github.com/networktocode/ntc-templates/blob/v3.1.0/pyproject.toml
